locales/ko/common.json

```diff
@@ -0,0 +1,24 @@
+{
+    "newsletterCTA": {
```

Comment:
Hey @mirusu400, there are some vulnerabilities in your pull request. kindly try to imitate the content with the original one [here](https://github.com/asyncapi/website/blob/master/locales/en/common.json). Moreover, some of the properties are missing. please consider them as well.
![image](https://github.com/asyncapi/website/assets/96991785/6df9863e-6623-46f5-b700-2b5b8e786b1a)